packages/xsnap/src/xsnap.c

```diff
@@ -859,11 +860,13 @@ void fxFreezeBuiltIns(txMachine* the)
 	mxPop();
 }
 
+#if 0
```

Comment:
Is this permanently dead code? How do we decide whether to remove the pragma fence or remove the code?

Reply:
Again, it's good that you call my bluff here.

This is the [how to organize xsnap?](https://github.com/endojs/endo/issues/681) issue, to my mind. I lean toward a separate xsnap-native submodule.

I guess I might as well delete this code rather than `#if 0` it, but I wanted to prompt some discussion first.

Reply:
fixed in e432531